scripts/multisrc/lightnovelwp/template.ts

```diff
@@ -249,10 +252,13 @@ class LightNovelWPPlugin implements Plugin.PluginBase {
 
   async parseChapter(chapterPath: string): Promise<string> {
     const $ = await this.getCheerio(this.site + chapterPath, false);
-    if (this.id == 'kolnovel') {
-      let ignore = $('article > style').text().trim().split(',');
-      ignore.push(...(ignore.pop()?.match(/^\.\w+/) || []));
-      ignore.map(tag => $(`p${tag}`).remove());
+    if (this.options && this.options.customJs) {
+      try {
+        eval(this.options.customJs);
```

Comment:
Instead of using eval function which is slow and unsafe, you can write the customJs directly.
I suggest adding some comment like `//~~replace me nii~~`, then replace them by customeJs in generator